apps/src/templates/studioHomepages/CourseBlocks.jsx

```diff
@@ -187,16 +171,24 @@ export const CourseBlocksAll = React.createClass({
         {!this.props.isEnglish && (
           <ProtectedStatefulDiv ref="csfCoursesHeader"/>
         )}
-        <CourseBlocksCsf isEnglish={this.props.isEnglish}/>
 
```

Comment:
Is it preferred to have something like ` const { isEnglish, isRtl, codeOrgUrlPrefix } = this.props; ` so you don't have to use `this.props.isEnglish` is other places? Or does it not matter?